java-hello-world-maven/README-Cloud-Shell.md

```diff
@@ -147,15 +166,27 @@ This step is optional - [Check software version and environment variables](./REA
     ...
     ```
 
-    4.2) **Option 2: Quick Build disabled** 
+3. Run the native executable using:
+
+    ```shell
+    ./target/my-app
+    ```
+
+    The output should be similar to:
+    ```
+    Hello World!
+    ```
```

Comment:
Why do you leave empty lines?

Reply:
Empty lines is for better readability. Whitespace is easier on the eyes. :-)